tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es6",
+        "target": "es2018",
```

Comment:
This makes compilation successful fixing the error above, but I have no idea about the downsides of changing this.  @tim-band do you happen to know more about this?